src/toolkit/gis.h

```diff
@@ -0,0 +1,158 @@
+#ifndef CYCLUS_SRC_GIS_H_
+#define CYCLUS_SRC_GIS_H_
+
+#include <math.h>
+#include <stdio.h>
+#include <sstream>
+#include <string>
+
+#include "cyclus.h"
+
+// forward declarations
+namespace cyclus {
+namespace toolkt {
+class GIS;
+}
+}
+
+namespace cyclus {
+namespace toolkit {
+/// @class GIS
+/// The GIS class is a basic class that stores the geographic location of each
+/// agent in latitude and longitude and follows the ISO
+/// 6709 standard. Allows string expression of GIS objects as specified in
+/// ANNEX H of ISO 6709 found here: 'https://www.iso.org/standard/39242.html'
+/// or a quick simplified version found here
+/// 'https://en.wikipedia.org/wiki/ISO_6709'
+///
+/// Brief Description of ISO 6709:
+/// 1. Latitude comes before Longitude
+/// 2. North latitude is positive
+/// 3. East longitude is positive
+/// 4. Fraction of degrees (decimal values) is prefered in digital data
+/// exchange.
+///
+/// The class is adapted from 'https://github.com/jaime-olivares/coordinate'
+/// under the MIT License.
+///
+/// Longitude and Latitude is stored as seconds of degrees. This was also
+/// adapted from 'https://github.com/jaime-olivares/coordinate'. This, according
+/// to the jaime-olivares, allows the coordinate elements such as degrees,
+/// minutes, and seconds to "remain on the integral portion of values, with the
+/// exception of decimal of seconds, avoiding loss of precision." This is
+/// calculated by multiplying decimal degrees by 3600.
+/// example: 05.2169 -> 18780.84
+
+class GIS {
+ public:
+  /// The default constructor for GIS. This will creat an object with
+  /// lat =0, and long=0.
+  GIS();
+
+  /// GIS constructor with latitude and longditude as degrees as a decimal
+  /// number.
+  /// @param deciaml_lat latitude expressed in degrees as a decimal number.
+  /// @param decimal_lon longditude expressed in degrees as a decimal number.
+  GIS(float decimal_lat, float decimal_lon);
```

Comment:
I think float should have enough precision for this case but I may be wrong. Originally, I decided to store latitude and longitude as float as for the same reason that jaime-olivares, has:

> The coordinate components will be stored as "seconds of degree" in float variables. This way, all coordinate elements (degrees, minutes, and seconds) will remain on the integral portion of values, with the exception of decimal of seconds, avoiding loss of precision. 
(https://github.com/jaime-olivares/coordinate)

The webpage also has a rationale against using float rather than double.

So, I think float should provide enough precision for this task. However, if I am wrong, please let me know and I will fix that right away.